composer.json

```diff
@@ -16,18 +16,18 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "symfony/framework-bundle": "^5.4 || ^6.0 ",
-        "symfony/intl": "^5.4 || ^6.0",
-        "symfony/templating": "^5.4  || ^6.0",
-        "symfony/validator": "^5.4  || ^6.0",
-        "symfony/yaml": "^5.4  || ^6.0",
+        "symfony/framework-bundle": "^5.4 || ^6.0 || ^7.0 ",
+        "symfony/intl": "^5.4 || ^6.0 || ^7.0 ",
+        "symfony/templating": "^5.4  || ^6.0 || ^7.0 ",
+        "symfony/validator": "^5.4  || ^6.0 || ^7.0 ",
+        "symfony/yaml": "^5.4  || ^6.0 || ^7.0 ",
         "psr/log": "^1.0 || ^2.0 || ^3.0"
     },
     "require-dev": {
         "symfony/phpunit-bridge": ">=5.4",
```

Comment:
older bridges have some issue. as this is only a dependency to run the testing, we can go with the very latest version for all tests:

```suggestion
        "symfony/phpunit-bridge": ^7.0.3",
```